x-pack/plugins/fleet/public/applications/integrations/sections/epm/screens/detail/assets/assets.tsx

```diff
@@ -102,31 +104,49 @@ export const AssetsPage = ({ packageInfo }: AssetsPanelProps) => {
       />
     );
   } else if (assetSavedObjects === undefined) {
-    content = (
-      <EuiTitle>
-        <h2>
-          <FormattedMessage
-            id="xpack.fleet.epm.packageDetails.assets.noAssetsFoundLabel"
-            defaultMessage="No assets found"
-          />
-        </h2>
-      </EuiTitle>
-    );
+    if (CustomAssets) {
+      // If a UI extension for custom asset entries is defined, render the custom component here depisite
+      // there being no saved objects found
+      content = (
+        <ExtensionWrapper>
+          <CustomAssets pkgkey={pkgkey} packageInfo={packageInfo} />
+        </ExtensionWrapper>
+      );
+    } else {
+      content = (
+        <EuiTitle>
+          <h2>
+            <FormattedMessage
+              id="xpack.fleet.epm.packageDetails.assets.noAssetsFoundLabel"
+              defaultMessage="No assets found"
+            />
+          </h2>
+        </EuiTitle>
+      );
+    }
   } else {
-    content = allowedAssetTypes.map((assetType) => {
-      const sectionAssetSavedObjects = assetSavedObjects.filter((so) => so.type === assetType);
+    content = [
+      ...allowedAssetTypes.map((assetType) => {
+        const sectionAssetSavedObjects = assetSavedObjects.filter((so) => so.type === assetType);
 
-      if (!sectionAssetSavedObjects.length) {
-        return null;
-      }
+        if (!sectionAssetSavedObjects.length) {
+          return null;
+        }
 
-      return (
-        <>
-          <AssetsAccordion savedObjects={sectionAssetSavedObjects} type={assetType} />
-          <EuiSpacer size="l" />
-        </>
-      );
-    });
+        return (
+          <>
+            <AssetsAccordion savedObjects={sectionAssetSavedObjects} type={assetType} />
+            <EuiSpacer size="l" />
+          </>
+        );
+      }),
+      // Ensure we add any custom assets provided via UI extension to the end of the list of other assets
+      CustomAssets ? (
+        <ExtensionWrapper>
+          <CustomAssets pkgkey={pkgkey} packageInfo={packageInfo} />
```

Comment:
can we expand the first asset accordion on this page by default? i.e. expand "Views" by default for these asset-less integrations